src/compositions/filters.ts

```diff
@@ -631,6 +632,12 @@ export function useRecentFlowRunsFilterFromRoute(defaultValue: MaybeReactive<Flo
   return response
 }
 
+const customDefaultFlowRunsFilterKey = getCacheKey('prefect-ui-library-custom-default-flow-runs-filter')
+
+export function useCustomDefaultFlowRunsFilter(): ReturnType<typeof useLocalStorage<FlowRunsFilter | null>> {
+  return useLocalStorage<FlowRunsFilter>(customDefaultFlowRunsFilterKey)
+}
+
```

Comment:
nit: this seems specific to the flow runs page vs the rest of this file which is generic of implementation. I'd move this to a new file.